addons/hide-new-variables/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Hide new variables",
-  "description": "Do not automatically create monitors for newly created variables or lists.",
+  "description": "Do not automatically show new variables or lists on the stage.",
```

Comment:
So are we going to avoid using the word "monitor"?

Reply:
I don't think we need to, but I think this description is fine.

Reply:
Here are a few variations that use the word “monitors”:
```suggestion
  "description": "Do not automatically show variable and list monitors on the stage.",
```
```suggestion
  "description": "Do not automatically show monitors for new variables or lists.",
```
```suggestion
  "description": "Do not automatically show monitors for newly created variables or lists.",
```

Or one other option:
```suggestion
  "description": "Hide newly created variables and lists by default.",
```

Reply:
I like the third one best, as it's clear and uses 'monitor.'

Reply:
I didn't mean to say we should be using the word "monitor". Honestly, I think we should not